tests/manage/rgw/test_multipart_upload.py

```diff
@@ -106,7 +106,10 @@ def test_multipart_upload_operations(
         logger.info(
             "Downloading the completed multipart object from the RGW bucket to the awscli pod"
         )
-        sync_object_directory(awscli_pod_session, object_path, res_dir, bucket)
+        awscli_pod_session.exec_sh_cmd_on_pod(command="apk add s3cmd")
```

Comment:
We may need to update the image to have s3cmd pre-configured. When we scale the tests for any negative scenarios, this configuration may be lost. 
This could be done in a different PR.

Reply:
@keesturam I have created issue to address this
https://github.com/red-hat-storage/ocs-ci/issues/9181